playbooks/openshift/README.md

```diff
@@ -93,6 +94,34 @@ For automatic, self-provisioned app routes to work, you will need a wildcard DNS
  
 In general, see https://docs.openshift.org/latest/install_config/install/prerequisites.html
 
+#### Alternative 2: containerized environment for Heat based deployment
+
+We also have a deployment container with all dependencies preinstalled. To build the container, 
+check out PAC git repo (see below) and run the build script located in `container-src/pac-deployer`:
+ 
+    cd ~/git/pouta-ansible-cluster/container-src/pac-deployer 
+    ./build.bash
+    
+To launch a shell in a temporary container for deployment, run    
+    
+    cd ~/git
```

Comment:
Made a comment on using sudo if necessary for both build and run phases. Added a helper script.